etl/scripts-ccao-data-warehouse-us-east-1/ccao/ccao-legacy.R

```diff
@@ -99,6 +99,7 @@ cc_dli_senfrr <- map_dfr(files_cc_dli_senfrr$Key, \(f) {
 
 # Write the files to S3, partitioned by year
 cc_dli_senfrr %>%
+  mutate(loaded_at = as.character(Sys.time())) %>%
```

Comment:
@jeancochrane My thinking here is that it would be nice and convenient to have a `loaded_at` column on every Athena source so that we can just `SELECT MAX(loaded_at) FROM $TABLE` for everything in our catalog and know what hasn't been touched in awhile.

Reply:
Right @dfsnow, I was just a bit confused by this one since it seems categorically different from our other sources (one-time ingest rather than something we pull regularly). But if you think it's easier for everything to follow the same pattern it's fine by me.